packages/trpc/server/routers/viewer/organizations/updateUser.handler.ts

```diff
@@ -72,6 +90,21 @@ export const updateUserHandler = async ({ ctx, input }: UpdateUserOptions) => {
     }),
   ]);
 
+  if (input.role === MembershipRole.ADMIN || input.role === MembershipRole.OWNER) {
```

Comment:
When we make someone an admin in the organization, let's also make them an admin in all the teams they're part of. But, only do this when they go from being a regular member to an admin, We don't want to accidentally strip admin privileges of this user from sub-teams if we have explicitly defined it

[Screencast from 10-12-23 02:14:02 PM IST.webm](https://github.com/calcom/cal.com/assets/32706411/49c4400e-ef49-4145-9ea2-092eb18667cb)